src/utils/FillUtils.ts

```diff
@@ -67,25 +67,27 @@ export async function getUnfilledDeposits(
   depositLookBack?: number
 ): Promise<{ [chainId: number]: RelayerUnfilledDeposit[] }> {
   const unfilledDeposits: { [chainId: number]: RelayerUnfilledDeposit[] } = {};
-  const chainIds = Object.values(spokePoolClients).map(({ chainId }) => chainId);
+  const chainIds = Object.values(spokePoolClients)
+    .filter(({ isUpdated }) => isUpdated)
+    .map(({ chainId }) => chainId);
```

Comment:
Should we enforce the numeric type of chainId? It would implicitly type the `chainIds`.
```suggestion
    .map(({ chainId }) => Number(chainId));
```

Reply:
@james-a-morris `SpokePoolClient.chainId` is explicitly defined as a number, so per current implementation it's not necessary and chainIds should be `number[]`. Is the suggestion to protect against `SpokePoolClient.chainId` changing type?